.kotlin_alpha/usecases/first_android_app/Readme.md

```diff
@@ -0,0 +1,423 @@
+* Execute queries
+
+The following shows the application you'll create.
+
+![AWS Blog Application](images/app0.png)
+
+When you choose **Submit**, the data is persisted into an Amazon DynamoDB table named **Android**, as shown in the following illustration.
+
+![AWS Blog Application](images/table1.png)
+
+After the table is updated with a new item, a text message is sent to notify a mobile user who monitors the submissions.
+
+![AWS Blog Application](images/text1.png)
+
+This tutorial guides you through creating a sample Native Android application that invokes Amazon DynamoDB and Amazon Simple Notification Service (SNS) by using the AWS SDK for Kotlin. 
+
+#### Topics
+
++ Prerequisites
++ Create an Android project named **AWSAndroid**
++ Add the dependencies to your Andorid project
++ Create the layout XML file for your Android project
++ Create the Kotlin classes for your Android project
++ Run the Android application
+
+## Prerequisites
+
+To complete the tutorial, you need the following:
+
++ An AWS account
++ An IDE that lets you build Android projects (this example uses IntelliJ and the Android plugin. See [Android Studio Plugin Development](https://plugins.jetbrains.com/docs/intellij/android-studio.html) )
++ Java 1.8 SDK
++ Gradle 6.8 or higher
++ Min API version is 24
++ Requires core library desugaring in the Gradle build file
++ Source/target compat of 1.8
+
+### Important
+
++ The AWS services included in this document are included in the [AWS Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
++  This code has not been tested in all AWS Regions. Some AWS services are available only in specific regions. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services). 
++ Running this code might result in charges to your AWS account. 
++ Be sure to terminate all of the resources you create while going through this tutorial to ensure that you’re not charged.
+
+### Creating the resources
+
+Create an Amazon DynamoDB table named **Android** that contains a partition key named **id**. For information about creating an Amazon DynamoDB table, see [Create a Table](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-1.html). 
+
+
+## Create an Android project named AWSAndroid
+The first step is to create an Android project.
+
+![AWS Blog Application](images/project1.png)
+
+1. In the IntelliJ IDE, choose **File**, **New**, **Project**.
+2. In the **New Project** dialog box, choose **Android**.
+3. Choose **Empty Activity**.
+4. Choose **Next**.
+5. In the **Name** field, enter **AWSAndroid**.
+6. In the **Package name** field, enter **com.example.aws**. 
+7. From the **Language** field, choose **Kotlin**. 
+8. In the **Minimum API** field, specify 26.
+9. Choose **Finish**.
+
+## Add the dependencies to your Andorid project
```

Comment:
Android

Reply:
Fixed

Reply:
That's a good change as well, but I meant the title "Android project" has a typo.